geonotebook/kernel.py

```diff
@@ -365,17 +371,17 @@ def _add_layer(layer_name):
             params['zIndex'] = len(self.layers)
 
             cb = self._remote.add_wms_layer(layer.name, layer.vis_url, params)\
-                .then(_add_layer, self.rpc_error)
```

Comment:
@danlamanna wouldn't it be possible to have a method that can avoid this code duplication of attaching error handler specially for add layer calls?